src/main/target/AIRF7/target.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "AIF7"
+#define USBD_PRODUCT_STRING     "Racerstar AirF7"
```

Comment:
This product name seems to be inconsistent with a `manufacturer_id` of `AIRB`. Can you explain who will be selling and supporting this target?

Reply:
It is made by Airbot and will be sold via Banggood under their branding "Racestar"

Reply:
So who will be providing support / instructions for this board?

Reply:
Airbot does the design and manufacturing as a service for lots of the boards sold under different brands. In all those cases the boards aren't marketed as "Airbot" branded boards.

Reply:
@etracer65: What I am trying to get the ground of is the apparent discrepancy between the USB device name and the `manufacturer_id`, which is `AIRB`. I suspect that this will lead to user confusion.

Reply:
> What I am trying to get the ground of is the apparent discrepancy between the USB device name and the manufacturer_id, which is AIRB. I suspect that this will lead to user confusion.

I agree. My point was that Airbot manufactures lots of board for other brands and none of those boards are branded with Airbot - so this is very unusual. I don't think the manufacturer ID should be `AIRB` and I pointed this out in earlier comments.

Reply:
I think we can use Racestar 's manufacturer ID for this target, that makes sense too because it is under their branding.
For the TARGET_BOARD_IDENTIFIER, I think it should be RacestarAirF7 too. 
With Racestar in front of everything, it should be clear enough for the user.

Reply:
@AirbotFPV: #8807. I think making this consistent will help both you and us to minimise user support issues.

Reply:
Done.

Reply:
Thanks